### PR TITLE
added basic map

### PR DESCRIPTION
**Pull request by miahsanchez**
I followed the example from https://www.digitalocean.com/community/tutorials/how-to-integrate-the-google-maps-api-into-react-applications to create a basic google map that shows up on the page and is centered at a random location. No styling has been done yet !

### CODE REVIEW

File: project/frontend/src/ResultsMap.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Map, GoogleApiWrapper } from 'google-maps-react';
+import React from 'react';
+
+const mapStyle = {
+  border: 'thin solid black',
+  height: '500px',
+  width: '500px',
+};
+
+export class MapContainer extends React.Component {
```
Comment (surbhimadan1995) [inline project/frontend/src/ResultsMap.js:10]: Lets add aria labels to this as we go: https://reactjs.org/docs/accessibility.html

Comment (miahsanchez) [reply project/frontend/src/ResultsMap.js:10]: add the aria label to the class ?

**Comment by miahsanchez** [comment]
UPDATE: I changed this from a class to a functional component :)

File: project/frontend/src/ResultsMap.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  border: 'thin solid black',
+  height: '500px',
+  width: '500px',
+};
+
+function MapContainer(props) {
+  const coords = { lat: 40.837, lng: -73.865 };
+  return (
+    <Map
+      ari-label={'A Google Map with your Matches!'}
```
Comment (surbhimadan1995) [inline project/frontend/src/ResultsMap.js:14]: should this be "aria" ?

**Pull request merged**